electricitymap/contrib/capacity_parsers/EIA.py

```diff
@@ -0,0 +1,100 @@
+import calendar
+from datetime import datetime
+from logging import getLogger
+from typing import Dict, Union
+
+import pandas as pd
+from requests import Response, Session
+
+from electricitymap.contrib.config import ZoneKey
+from parsers.EIA import REGIONS
+from parsers.lib.utils import get_token
+
+logger = getLogger(__name__)
+
+CAPACITY_URL = "https://api.eia.gov/v2/electricity/operating-generator-capacity/data/?frequency=monthly&data[0]=nameplate-capacity-mw&facets[balancing_authority_code][]={}"
+API_KEY = get_token("EIA_KEY")
+
+US_ZONES = {key: value for key, value in REGIONS.items() if key.startswith("US-")}
+TECHNOLOGY_TO_MODE = {
```

Comment:
Can't we have that from the normal EIA parser?

Reply:
<img width="426" alt="image" src="https://github.com/electricitymaps/electricitymaps-contrib/assets/107848894/57a42768-697f-4668-be01-2c5d732da2c5">
It is not the same mapping as it is already aggregated for the hourly generation route. 
I did the mapping using the technology rather than the code because i think it is more comprehensive and easier to spot if i made a mistake